girder/api/v1/user.py

```diff
@@ -196,6 +197,14 @@ def deleteUser(self, user):
         self.model('user').remove(user)
         return {'message': 'Deleted user %s.' % user['login']}
 
+    @access.admin
+    @autoDescribeRoute(
+        Description('Get the number of users.')
+    )
+    def getUsersDetails(self):
+        nUsers = self.model('user').collection.find().count()
```

Comment:
This should be:
```
self.model('user').find().count()
```
The [Model itself has a `find` method](https://github.com/girder/girder/blob/master/girder/models/model_base.py#L237), which returns a PyMongo cursor, so we don't need to go to the underlying PyMongo collection to call the method.